cmd/root.go

```diff
@@ -68,23 +75,25 @@ func init() {
 	// TODO() Publish command, not completed
 	// TODO() Report command, not completed
 	// TODO() have a single function that conditionally (i.e., Trace=true) prints ALL Flags
-	RootCmd.PersistentFlags().StringVar(&utils.Flags.CfgFile, "config", "", wski18n.T(wski18n.ID_CMD_FLAG_CONFIG))
```

Comment:
Converted all hard coded strings to CONST, for example, "config" to `FLAG_CONFIG`, etc